frontend/src/components/common/table/SearchableTablePage.tsx

```diff
@@ -37,13 +37,11 @@ const SearchableTablePage = <T, SortPropTypes extends readonly string[]>({
   return (
     <>
       <VStack pt={4} spacing={6} w="full">
-        {searchLength !== 0 && (
```

Comment:
Any chance we could rename this to something like `resultsLength`? `searchLength` makes it sound like the length of the search string itself to me